packages/ovh-manager/server/proxy/Aapi/index.js

```diff
@@ -0,0 +1,17 @@
+const config = {
+  aapi: {
+    url: 'http://localhost:8080',
+  },
+};
+
+module.exports = {
+  context: ['/engine/2api', '/2api'],
```

Comment:
why declare 2 paths `/engine/2api` and `2api`, we should only use `/engine/2api` (as production, no ?